### PR TITLE
Feat/create mock data

### PR DESCRIPTION
**Pull request by hirichardhai**
finished seeding 10 million similar listings. can you guys please look over this? thanks.

### CODE REVIEW

**Comment by waynekwest88** [comment]
Looks good!

File: db/SDC/joshClusterVersion.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const generateOneSimilarListing = require('./haiDataGenerator').generateOneSimilarListing;
+const MongoClient = require('mongodb').MongoClient;
```
Comment (zinoki) [inline db/SDC/joshClusterVersion.js]: could use object destructuring instead when using require

File: db/SDC/joshClusterVersion.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const generateOneSimilarListing = require('./haiDataGenerator').generateOneSimilarListing;
```
Comment (zinoki) [inline db/SDC/joshClusterVersion.js]: could use object destructuring instead when using require

**Comment by zinoki** [comment]
looks good

**Pull request merged**